website/addons/wiki/views.py

```diff
@@ -236,8 +239,11 @@ def project_wiki_view(auth, wname, path=None, **kwargs):
     wiki_name = (wname or '').strip()
     wiki_key = to_mongo_key(wiki_name)
     wiki_page = node.get_wiki_page(wiki_name)
+    wiki_settings = node.get_addon('wiki')
     toc = _serialize_wiki_toc(node, auth=auth)
-    can_edit = node.has_permission(auth.user, 'write') and not node.is_registration
+    can_edit = auth.logged_in and not node.is_registration and\
```

Comment:
Remove this line continuation. The following is equivalent:

```python
can_edit = (
    auth.logged_in
    and not node.is_registration
    and (
        node.has_permission(auth.user, 'write')
        or wiki_settings.is_publicly_editable
    )
)
```

Whenever possible, parenthesis should be used for continuations, not `\`.